src/lib/components/media/riven/item-manual-scrape.svelte

```
@@ -0,0 +1,867 @@
+    let selectedOptions = $state<Record<string, string[]>>({
+        resolutions: [],
+        quality: [],
+        rips: [],
+        hdr: [],
+        audio: [],
+        extras: [],
+        trash: []
+    });
+    let canStartAutoScrape = $derived(Object.values(selectedOptions).some(arr => arr.length > 0));
+
+    const categoryIcons: Record<string, any> = {
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)